src/lib.rs

```diff
@@ -3233,6 +3233,11 @@ impl Connection {
             .is_none()
     }
 
+    /// Returns whether there is stream or DATAGRAM data available to read.
+    pub fn is_app_data_readable(&self) -> bool {
```

Comment:
I think we can just call this `is_readable()` to be consistent with `readable()` (and also app data is really the only thing that can be read from the connectio anyway).